### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Anton-Zhytonbaiev**
[DEMO LINK](https://Anton-Zhytonbaiev.github.io/react_sum/)

### CODE REVIEW

File: src/components/Sum.jsx [hunk 1]
```diff
@@ -1 +1,11 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```
Comment (vanst1) [inline src/components/Sum.jsx:4]:
```suggestion
  <p>
    {`Sum of ${a} and ${b} is ${a + b}`}
  </p>
```

**Review by vanst1 — APPROVED** [review]
Good job)

File: src/components/Sum.jsx [hunk 2]
```diff
@@ -1 +1,11 @@
...
+);
+
+// <p>Sum of 2 and 3 is 5</p>
```
Comment (alexandra-protyanova) [inline src/components/Sum.jsx:7]: remove comments

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
Add prop-types

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
In previous review I asked to add prop-types)

**Review by alexandra-protyanova — APPROVED** [review]
GJ 🔥

**Pull request closed without merging**